inspirehep/dojson/hep/fields/bd20x24x.py

```diff
@@ -32,7 +32,6 @@
 
 
 @hep.over('title_translations', '^242[10_][0_]')
```

Comment:
I learn from @annetteholtkamp that the main title (245) is not necessarily the foreign language when a language for a document is set.